README.md

```diff
@@ -6,3 +6,6 @@
 - [iOS](van-sdk/ios.md)
 - [iOS (Korean) ](van-sdk/ios_kr.md)
 - [S2S](van-sdk/s2s.md)
+
+### Android SDK 1.0
```

Comment:
`Android SDK` looks ambiguous in this space. Let's use something like `Ad Serving SDK` and use prefix of `Android (English)` or just `Android` below.